components/domains/externalDomainActions.tsx

```diff
@@ -51,27 +56,67 @@ const ExternalDomainActions: FunctionComponent<ExternalDomainActionsProps> = ({
     set_address_to_domain();
   }
 
+  function getResolverContract(): string {
```

Comment:
to put somehwere in utils and test. 

We'll reuse this more and more as we have more external subdomains being built. 

Maybe you can even add it to starknetid.js to make it available for everyone (up to you, if you find this relevant enough)